uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/manager-leave/components/manager-leave-requests_test.js

```diff
@@ -147,8 +149,17 @@
             expect(controller.levelTypes).toEqual(optionGroupMock.getCollection('hrjc_level_type'));
           });
 
-          it('absence periods have loaded', function () {
-            expect(controller.absencePeriods.length).not.toBe(0);
+          describe('absence periods', function() {
+            it('absence periods have loaded', function () {
```

Comment:
always try to makes a sentence with `it`, to improve readibility of the tests
```js
it('loads the absence periods'
```
?